ext/import-font.cpp

```diff
@@ -98,10 +98,16 @@ bool loadGlyph(Shape &output, FontHandle *font, int unicode, double *advance) {
     FT_Error error = FT_Load_Char(font->face, unicode, FT_LOAD_NO_SCALE);
     if (error)
         return false;
+
+    double unitsPerEm;
+    double unitScale;
+    getFontScale(unitsPerEm, font);
+    unitScale = 2048.0 / unitsPerEm;
```

Comment:
Why is there a 2048 hardcoded here?  This definitely looks wrong to me.  Looks like there's a 2048 assumption baked into the code somewhere else, and the math here will undo that.  Proper fix is to replace all 2048's in the code with unitsPerEm instead.